extensions/wikia/EditPageLayout/js/editor/AceEditor.js

```diff
@@ -176,6 +177,19 @@ define(
 		$('.editpage-widemode-trigger').click(editorModeChange);
 	}
 
+	function launchEditorPreview(){
+		var editorValue = ace.getContent(),
+			wikiaDomain = mw.config.get('wgServer').split('://')[1],
+			templateName = mw.Title(mw.config.get('wgPageName'), 'Template').title,
```

Comment:
@katanka mw.Title is class, isn't it? So should be used with new ```new``` keyword.
Would that hardcoded "Template" work for all languages?
Use getters instead of accessing fields directly.
It should be done more or less like this: ```(new mw.Title("Template:123/456")).getMain()``` (not sure about getMain method - maybe some other method should be used)